packages/cojson/src/coValue.ts

```diff
@@ -18,7 +15,7 @@ export type CoID<T extends RawCoValue> = RawCoID & {
 export interface RawCoValue {
   /** The `CoValue`'s (precisely typed) `CoID` */
   id: CoID<this>;
-  core: AvailableCoValueCore;
+  core: CoValueCoreVerified;
```

Comment:
Renamed because it isn't related to Available state, which requires also the CoValue deps, but only a CoValue with a VerifiedState